lib/ci-helper.ts

```diff
@@ -319,30 +319,30 @@ export class CIHelper {
         const maintainerBranch = `refs/remotes/${this.config.repo.maintainerBranch}/`;
         const maintainerRepo = `${this.config.repo.maintainerBranch}.${this.config.repo.name}`;
 
-        let gitsterBranch: string | undefined =
+        let maintainerBranch: string | undefined =
```

Comment:
FWIW the variable named `maintainerBranch` has already been declared 3 lines above, but it refers to the `refs/remotes/gitster/` prefix.

That is the culprit why all CI builds fail, see e.g. https://github.com/gitgitgadget/gitgitgadget/actions/runs/3345902406/jobs/5542023393#step:9:16